### ISSUE TITLE
Isracard scraping doesn't work

### ISSUE THREAD

**Issue by erikash** [eshaham/israeli-bank-scrapers#486]
Hi,
I tried to scrape Isracard with my friends credentials (I don't have then any longer so I can't reproduce)
Puppeteer couldn't fill in in the 6digits + id + password (couldn't see the details being filled).
I scraped successfully using the same environment `hapoalim` account so it's not an evironment issue.

Using Node.js version `10.22` and `israeli-bank-scrapers` `0.9.1`.

The error I got was `INVALID_PASSWORD`.

Sorry for missing additional details.. can anyone please scrape an Israecard account to reproduce?

Thanks,
Erik.

**Comment by baruchiro** [comment]
It just said you have the wrong credentials.

And you don't see anything because it works with HTTP Requests.

**Comment by erikash** [comment]
Thanks for the reply Baruch.
I’ve logged in with the credentials successfully manually...

Did you have a chance to test the integration?

**Comment by baruchiro** [comment]
Yes, it worked for me today.

**Comment by esakal** [comment]
@erikash Isracard seems to work for me as well. I know it might be obvious but make sure you set the `password` and the `card6Digits` values correctly. I spent a few hour debugging just to figure out that I switched those values.

I was using both node 12.x and 8.x so it doesn't seem to be related to node version.

If you cannot reproduce it let's close this issue and once you will have additional information just comment again in this issue.
